### DISCUSSION TITLE
What generates the font-family for the `body`? Doesn't seem to honour my tailwind.config

### DISCUSSION THREAD

**Discussion by quagliero** [tailwindlabs/tailwindcss#3424] [Help] [answered]
Hi, I'm running a nextjs project and I'm setting font families in the theme section of the tailwind.config like so:
```js
  theme: {
    fontFamily: {
      sans: 'Inter,-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Open Sans","Helvetica Neue",sans-serif',
      serif: 'Inter,-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Open Sans","Helvetica Neue",sans-serif',
      body: 'Inter,-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Open Sans","Helvetica Neue",sans-serif',
      display: 'Inter,-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Open Sans","Helvetica Neue",sans-serif',
    },
```

And including the CSS like so:
```
@tailwind base;
@tailwind components;
@tailwind utilities;
```

Yet there is something setting the font on the `body` that is ignoring these config options. I looked through the [source code](https://github.com/tailwindlabs/tailwindcss/blob/9585ae15a3dd216c1e399ffad2fe6755ee02e215/src/plugins/css/preflight.css) and saw that the html font-family is set during `preflight.css`, but can't find any reference to where the `font-family` on the `body` is being set. Therefore the body is overiding the html style and the font just keeps using the system default (image below).

<img width="330" alt="Screenshot 2021-01-23 at 14 11 22" src="https://user-images.githubusercontent.com/4254570/105580554-eab92300-5d84-11eb-9210-2bed0dc0cf79.png">

Any help appreciated.

**Comment by adamwathan** [comment, marked as answer]
Hey! The font family on body is set to `inherit` as demonstrated in your screenshot, which will pull from the value on the `html` element. It should be your `Inter` based font stack. This is ultimately controlled by the `fontFamily.sans` setting in your theme 👍🏻 

What you are seeing there should be giving you the results you want, nothing is overriding your Inter font there. You can see what font is actually being rendered by looking at the _very_ bottom of the "Computed" style tab in dev tools:

<img width="371" alt="image" src="https://user-images.githubusercontent.com/4323180/105580848-e24cf200-5d5c-11eb-962c-fd9f673b5e2f.png">

**Reply by quagliero** [reply]
Thanks @adamwathan - you're absolutely right. I was being thrown off by a devtools red herring. Nice tip on the 'Rendered Fonts' section at the bottom of the computed tab. 👍🏻

**Comment by tsega** [comment]
I am using [Nuxt + Google Fonts](https://google-fonts.nuxtjs.org/) -  in my situation not importing the specific font weights resulted in the font not being used so I had to update my `nuxt.config.js` as follows:

```javascript
// https://google-fonts.nuxtjs.org/options
  googleFonts: {
    families: {
      Inter: [100, 400, 700, 900]
    }
  },
```
